### ISSUE TITLE
Extension not compatible with java8, built on level11

### ISSUE THREAD

**Issue by laurentperez** [quarkiverse/quarkus-jooq#21]
Hello

Quarkus 2 (alpha) requires JDK11, but JDK8 support is maintained for Quarkus 1.x

However the extension jar on maven central is compiled against a newer JDK, so it can't run under a level 8 project

`java.lang.RuntimeException: java.lang.UnsupportedClassVersionError: io/quarkiverse/jooq/runtime/JooqConfig has been compiled by a more recent version of the Java Runtime (class file version 53.0), this version of the Java Runtime only recognizes class file versions up to 52.0
`
We're using the early tag `implementation 'io.quarkiverse.jooq:quarkus-jooq:0.1.0'`

Is it possible for you to lower the version ?

**Comment by laurentperez** [comment]
I'm not sure what happened because https://github.com/quarkiverse/quarkus-jooq/blob/master/pom.xml#L24 is at level 1.8

But the quarkus-jooq-0.1.0.jar jar is 

```
Build-Jdk-Spec: 11
Specification-Title: Quarkus jOOQ - Runtime
Specification-Version: 0.1
Specification-Vendor: JBoss by Red Hat
Implementation-Title: Quarkus jOOQ - Runtime
Implementation-Version: 0.1.0
Implementation-Vendor: JBoss by Red Hat
Implementation-URL: https://quarkiverse.io/quarkus-jooq-parent/quarkus-j
 ooq
Java-Vendor: Azul Systems, Inc.
Java-Version: 11.0.10
Os-Arch: amd64
Os-Name: Linux
Os-Version: 5.4.0-1043-azure
```

**Comment by angrymango** [comment]
Fixed in release 0.2.0 - https://github.com/quarkiverse/quarkus-jooq/releases/tag/0.2.0
